concepts/strings/introduction.md

```diff
@@ -0,0 +1,29 @@
+# Introduction
+
+A `string` in Go is an immutable sequence of bytes, which don't necessarily have to represent characters.
+
+A `string` literal is a sequence of bytes enclosed in double quotes:
+
+```go
+name := "Jane"
+```
+
+Strings can be concatenated via the `+` operator:
+
+```go
+fullName := "Jane" + " " + "Austen"
+```
+
+Some special characters need to be escaped with a leading backspace:
+
+```go
+question := "\t or spaces?"
+```
+
+The `strings` package contains many useful functions to work on strings.
+
+```go
+import "strings"
+
+strings.ToUpper("test") // Output: "TEST"
+```
```

Comment:
Since there is another concept dedicated to this, should we put this here at all?

Reply:
We shouldn't! Copy-paste error.